services/QuillLessonsServer/bootstrap.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Installing script dependencies..."
+brew install jq
+
+echo "Installing RethinkDB for local use..."
+brew update
+brew install rethinkdb
+
+echo "Setting up local RethinkDB in prep to load seed data..."
+./rethink_local.sh start
+echo -n "Please provide the path to your RethinkDB seed data dump: "
+read rethink_dump_path
+echo "Loading seed data into local RethinkDB instance..."
+$(rethinkdb restore --force $rethink_dump_path)
+./rethink_local.sh stop
+
+echo "Generating and configuring JWT public/private keypair..."
+ssh-keygen -t rsa -b 4096 -N '' -m PEM -f lessons_server_jwt_keypair
+jq -n --arg cert "$(openssl rsa -in lessons_server_jwt_keypair -pubout)" '{env: {JWT_PUBLIC_KEY: $cert}}' > nodemon.json
+echo "LESSONS_PRIVATE_KEY=$(cat lessons_server_jwt_keypair | jq -R -c -s)" >> ../QuillLMS/.env
```

Comment:
Maybe this doesn't need to be printed to the screen?

Reply:
It isn't.  `echo` is used to send a string literal to STDOUT, but then the file append pipe `>>` intercepts and consumes it before it goes to the display.

Reply:
Ah, yeah, you are correct. It's been a little while since I've written bash, I'm a little rusty. 👍

Reply:
Well, I had to go test to make sure I was remembering correctly, so no worries.